graspologic/layouts/auto.py

```diff
@@ -200,11 +201,6 @@ def layout_umap(
     return lcc_graph, positions
 
 
-def _largest_connected_component(graph: nx.Graph) -> nx.Graph:
```

Comment:
Just a note that I used this because it was slightly faster than the overloaded largest_connected_component that has a bunch of conditional logic for different types.

I agree that it's an overoptimization that doesn't matter much, though